app/services/decorators/v5/capital_result_decorator.rb

```diff
@@ -17,7 +18,7 @@ def basic_attributes
         {
           total_liquid: summary.total_liquid.to_f,
           total_non_liquid: summary.total_non_liquid.to_f,
-          total_vehicle: summary.total_vehicle.to_f,
+          total_vehicle: @person_capital_subtotals&.total_vehicle.to_f,
```

Comment:
do we expect @person_capital_subtotals to ever be nil?

Reply:
Not if I make sure that there is always a PersonCapitalSubtotal object even if the person in question is a partner and the assessment has no partner. Which I think I should. So I can remove the & here. Thanks!